desc/compute/__init__.py

```diff
@@ -13,6 +13,15 @@
     compute_covariant_metric_coefficients,
     compute_contravariant_metric_coefficients,
     compute_geometry,
+    cross,
+    dot,
+)
+from .utils import (
```

Comment:
are these functions used anywhere outside the compute module? If not we probably don't want to expose them as part of the public api

Reply:
Oh if` desc/compute/__init__.py` is just for public api functions, then we probably don't want to add cross and dot. The ones from `.utils` could be useful.

Reply:
I removed the utils from the public API. This matches our usage in all the other modules. If someone really needs them, they can still access them.